public/app/plugins/panel/canvas/InlineEdit.tsx

```diff
@@ -20,16 +20,16 @@ const OFFSET_X = 10;
 const OFFSET_Y = 32;
 
 export function InlineEdit({ onClose, id, scene }: Props) {
-  const root = scene.root.div!.getBoundingClientRect();
+  const root = scene.root.div?.getBoundingClientRect();
   const windowHeight = window.innerHeight;
   const windowWidth = window.innerWidth;
   const ref = useRef<HTMLDivElement>(null);
   const styles = useStyles2(getStyles);
   const inlineEditKey = 'inlineEditPanel' + id.toString();
 
   const defaultMeasurements = { width: 350, height: 400 };
-  const defaultX = root.x + root.width - defaultMeasurements.width - OFFSET_X;
-  const defaultY = root.y + OFFSET_Y;
+  const defaultX = root?.x ?? 0 + (root?.width ?? 0) - defaultMeasurements.width - OFFSET_X;
```

Comment:
```suggestion
const defaultX =
    root?.x ?? 0 + (root?.width ?? defaultMeasurements.width + OFFSET_X * 2) - defaultMeasurements.width - OFFSET_X;
```

Reply:
Probably a simpler way to write this, but basically cancel out the negative values if root.width is undefined.